cloud-controller-manager/do/loadbalancers.go

```diff
@@ -335,6 +335,18 @@ func (l *loadBalancers) checkAndUpdateLBAndServiceCerts(ctx context.Context, ser
 		}
 	}
 
+serviceCheck:
```

Comment:
Should this be pulled out into a function and just `return l.checkServiceCertificate(...)` in both places?